Code/Framework/AzFramework/AzFramework/Physics/Configuration/JointConfiguration.h

```diff
@@ -66,4 +72,18 @@ namespace AzPhysics
         // Default all visibility settings to invisible, since most joint configurations don't need to display these.
         AZ::u8 m_propertyVisibilityFlags = 0;
     };
-}
+
+    //! Default colors, line widths etc for use when visualizing joints.
+    namespace JointVisualizationDefaults
+    {
+        const float Alpha = 0.6f;
+        const AZ::Color ColorDefault = AZ::Color(1.0f, 1.0f, 1.0f, Alpha);
+        const AZ::Color ColorFirst = AZ::Color(1.0f, 0.0f, 0.0f, Alpha);
+        const AZ::Color ColorSecond = AZ::Color(0.0f, 1.0f, 0.0f, Alpha);
+        const AZ::Color ColorSweepArc = AZ::Color(1.0f, 1.0f, 1.0f, Alpha);
+
+        const float SweepLineDisplaceFactor = 0.5f;
+        const float SweepLineThickness = 1.0f;
+        const float SweepLineGranularity = 1.0f;
```

Comment:
Since these are in a header now, you'd want to put `inline` in all of these so only 1 is used during linking.

Reply:
+1 (I'm actually surprised this compiles, if they were included in another file you'd likely get a compile error unfortunately 😛)

Reply:
Yeah, that is surprising - but it's actually included in a whole bunch of places and still compiles (and I don't think I'm building with unity on) so that's weird. In any case I'll add the inline.

Reply:
Actually, thinking about this some more, I think the compilation should be fine because of the pragma once.